packages/vapor/scss/common/palette.scss

```diff
@@ -70,7 +70,7 @@ $onahau-1: #b6eeff;
 $blue-purple-palette: category;
 $coveo-blue-purple: $mirage;
 $blue-purple-1: #3e5485;
-$blue-purple-2: #354878;
+$blue-purple-2: #354667;
```

Comment:
why this color has changed ?

Reply:
The color (`#354667`) was used in multiple places, but wasn't even in the color palette. So I replaced an existing color by it instead of creating yet another color (:wink: :wink: @oliviertaillon). The previous color (`#354878`) was only used for [the NumericInput](https://vapor.cloud.coveo.com/singleselect-custom-button/index.html#/components/NumericInput) [+] and [-] buttons.